src/app.ts

```diff
@@ -1,5 +1,5 @@
-import path from 'path'
-import fs from 'fs'
+import { readFileSync, existsSync } from 'fs'
+import { join } from 'path'
```

Comment:
Same as above.

Reply:
I can switch it back to `import * from`.  The tsconfig update caused synthetic default imports to no longer work.